main.js

```diff
@@ -18,4 +115,77 @@ mobileLinks.forEach(
       mobileMenu.style.display = 'none';
     });
   },
-);
\ No newline at end of file
+);
+
+const btnProjects = document.querySelectorAll('.btn-project');
+
+btnProjects.forEach(
+  (btnProject, i) => {
+    btnProject.addEventListener('click', () => {
+      listOfelements = '';
+      projects[i].technologies.forEach((technologie) => {
+        listOfelements += `<li class="card-tag"><span class="card-tag-text">${technologie}</span></li>`;
+      });
+      divPopup.classList.add('pop-up');
+      divPopup.innerHTML = ` <div class='pop-up-card'>
+<div class='pop-up-header'>
+    <div class='w-full'>
+        <h2>${projects[i].title}</h2>
+        <img id='pop-up-cross' src='resources/Normal Button/Tertiary/Icons/cross_black.png' alt='cross'>
+    </div>
+   
+    <div class='card-header-frame pop-up-header-span'>
+        <div class='card-header-frame-client'>
+            <span class='card-header-frame-client-text'>${projects[i].tags[0]}</span>
+        </div>
+        <span class='counter'>
+             <img src='resources/Counter.svg' alt='counter'> 
+        </span>
+        <div class='card-header-frame-role'>
+            <span class='card-header-frame-role-text'>${projects[i].tags[1]}</span>
+        </div>
+        <span class='counter'>
+             <img src='resources/Counter.svg' alt='counter'> 
+        </span>
+        <div class='card-header-frame-year'>
+            <span class='card-header-frame-year-text'>${projects[i].tags[2]}</span>
+        </div>
+    </div>
+</div>
+<img class='pop-up-img' src='${projects[i].imgsrc}' alt='fake2'>
+
+
+    <div class='pop-up-footer'>
+        <p>${projects[i].description.big}</p>
+      <div class='pop-up-resources'>
+        <ul class='card-tags'>
+
+        ${listOfelements}
+        </ul>
+        <div class='pop-up-buttons'>
+            <button type='button' class='button-link w-146 translate-button'><a target='_blank' href='${projects[i].links[0]}' class='button-link-text btn-flex'>See Live
+            <img class='p-3' src='resources/live_show.svg' alt='live_icon'>
+        </a>  </button>
+            <button type='button' class='button-link w-146 translate-button'><a target='_blank' href='${projects[i].links[1]}' class='button-link-text btn-flex'>See Source
+                <img class='p-3' src='resources/github_icon.svg' alt='live_icon'>
+            </a>
+            </button>
+
+        </div>
```

Comment:
- [Optional  ] Kindly adjust links for "See Live" and "See Source" for project, When I clicked on "See Live" it redirected me to GitHub but it must be to project live deploy and same for "See Source" it redirected me to project live deploy but it must be to git hub source code, Please fix it.

Reply:
There is no project for to fill so I will just add the current portfolio website